src/admin/dumb_components/letters/sub_letters_section.js

```diff
@@ -2,9 +2,11 @@ import React from 'react'
 import LetterRecipients from './letter_recipients.js'
 import StandingOrderLetter from './standing_order_letter.js'
 
-export default ({ letters, active_view }) => (
+export default ({ letters, toggle_recipient_list }) => (
   <div>
-    <button onClick={() => active_view('sub_reminders', !letters.sub_reminders.shown)}>
+    <button
+      className='letters-toggle'
+      onClick={() => toggle_recipient_list('sub_reminders', !letters.sub_reminders.shown)}>
```

Comment:
Some duplication between this and `post_members_seection`. I think it would be easy to resolve, but I'll merge this in as is for now